ui/shared/components/PopupWithModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { connect } from 'react-redux'
+import { Popup } from 'semantic-ui-react'
+
+import { getOpenModals } from 'redux/utils/modalReducer'
+
+const BehindModalPopup = styled(Popup)`
+  z-index: 500 !important;
+`
+
+class PopupWithModal extends React.PureComponent {
+
+  static propTypes = {
+    openModal: PropTypes.bool,
```

Comment:
why is the openModal prop necessary?

Reply:
1) this is state, not a prop
2) so that the popup won't close while the modal is open